tests/test_scavenging.py

```diff
@@ -541,3 +545,41 @@ def test_multiple_definition(v):
     check(v.defined_vars, ['a', 'a'])
     check(v.used_names, [])
     check(v.unused_vars, ['a', 'a'])
+
+
+@pytest.mark.skipif(sys.version_info < (3, 6),
+                    reason="requires python3.6 or higher")
+def test_typing_module(v):
+    v.scan("""\
+from typing import Iterable, List
+
+x: List[int] = [1]
+
+def f(n: int) -> Iterable[int]:
+    i = 0
+    while i < n:
+        yield i
+        i += 1
+""")
+
+    check(v.unused_vars, ['x'])
+    check(v.unused_funcs, ['f'])
+    check(v.used_names, ['Iterable', 'n', 'i', 'int', 'List'])
+
+
+@pytest.mark.skipif(sys.version_info < (3, 6),
```

Comment:
Hey, can we use [`pytest.importorskip`](https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency) instead of checking for Python version here?

Reply:
With this approach, Python 3.5 and 2.7 have their tests fail for me. Using `pytest.importorskip('typing')`

Reply:
Okay, no problem - let's merge this! :-)